connection/fncs_msg.cpp

```diff
@@ -1489,6 +1490,7 @@ static char fncs_unhex(char h)
 		return h-'A'+10;
 	else if ( h>='a' && h<='f' )
 		return h-'a'+10;
+  else return '?';
```

Comment:
I'm not super familiar with how FNCS handles these internally, so I'd recommend @afisher1 review this default return value, other than that, everything looks fine to me.